### DISCUSSION TITLE
What's your filtering strategy?

### DISCUSSION THREAD

**Discussion by wanderleyteixeira** [euzu/tuliprox#126] [General]
I was curious about how others are approaching filtering in `source.yml`. For instance, do you use broader filters like `Group ~ "UK..*"` to catch all `UK` groups and then rely on the Threadfin app to refine further and map channels for use in a streaming app?

Personally, I lean towards more specific filters for live channels, such as `Group ~ "^.UK..*" AND Name ~ "Sky News.*"`, to keep things targeted. However, I've noticed that my list tends to grow quite quickly this way and more changes every time I want to add a channel.

I have not even started with VOD or TV Series or Movies categories yet. 

Just thought this might make for an interesting discussion—curious to hear your thoughts!

**Comment by euzu** [comment]
@wanderleyteixeira I use templates to define each fine filter individually and then combine them to form larger filters until I achieve the final filter required for a target.

If you have any suggestions or ideas on how to make this process easier or more efficient, I’d be happy to hear them.
